src/test/java/woowacourse/shoppingcart/service/ProductServiceTest.java

```diff
@@ -0,0 +1,83 @@
+package woowacourse.shoppingcart.service;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.jdbc.Sql;
+import woowacourse.shoppingcart.dto.request.ProductRequestDto;
+import woowacourse.shoppingcart.dto.response.ProductResponseDto;
+import woowacourse.shoppingcart.exception.NotFoundProductException;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@Sql(scripts = {"classpath:schema-reset.sql"}, executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
+class ProductServiceTest {
+
+    @Autowired
+    private ProductService productService;
+    private ProductRequestDto product1 = new ProductRequestDto("product1", 10000, null, 10);
+    private ProductRequestDto product2 = new ProductRequestDto("product2", 11000, null, 10);
+
+    @Test
+    @DisplayName("상품 전체 목록을 불러온다.")
+    void findProducts() {
+        productService.addProduct(product1);
+        productService.addProduct(product2);
+
+        final List<ProductResponseDto> actual = productService.findProducts();
+
+        assertThat(actual.size()).isEqualTo(2);
+        assertThat(actual).extracting("name")
+                .contains("product1", "product2");
```

Comment:
```suggestion
        assertThat(actual).hasSize(2)
                .extracting("name")
                .contains("product1", "product2");
```

이렇게 체이닝 방식을 이용할 수도 있어요~